src/main/java/org/intellij/sonar/configuration/partials/SonarResourcesTableView.java

```diff
@@ -32,14 +33,18 @@
     @Nullable
     @Override
     public String valueOf(Resource sonarResource) {
-      if (Resource.QUALIFIER_PROJECT.equals(sonarResource.getQualifier())) {
-        return "Project";
-      } else
-        if (Resource.QUALIFIER_MODULE.equals(sonarResource.getQualifier())) {
+      SonarQualifier sonarQualifier = SonarQualifier.valueFrom(sonarResource.getQualifier());
+      if (sonarQualifier == null) {
```

Comment:
yes it should be null ;)

Reply:
As it throws an exception now, it should not, yes